staxapp/auth.py

```diff
@@ -139,7 +139,10 @@ def requests_auth(username, password, **kwargs):
 class ApiTokenAuth:
     @staticmethod
     def requests_auth(username, password, **kwargs):
-        if StaxConfig.expiration and StaxConfig.expiration > datetime.now(timezone.utc):
+        # Minimize the potentical for token to expire while still being used for auth (say within a lambda function)
+        if StaxConfig.expiration and StaxConfig.expiration - timedelta(
+            minutes=10
```

Comment:
I think 1 minute would be enough.

Reply:
Would be good to add this line to RootAuth section as well, just above it!

Reply:
@wolfeidau - where is `RootAuth` used? I don't see any references for it (except testing)

Reply:
@richardkeit it is a legacy thing, I need to check if we can remove it.